### PR TITLE
WB3000 public documentation 

### PR DESCRIPTION
**Pull request by ipanagiv**
Resolves FE-343, FE-348, FE-346, FE-990, FE-344, FE-345, FE-347, FE-1037, FE-1038, FE-1036

### CODE REVIEW

File: docs/wxm-devices/Pulse-bundle/Pulse-troubleshooting.mdx [hunk 1]
```diff
@@ -155,21 +94,17 @@ This incurs a "relocation penalty" (as mentioned in [the whitepaper](http://weat
...
-### What can I do if my WG1200 Gateway is switching off frequently?
-
-Please disconnect the WG1200 Gateway from the power source and plug it back to the power source. You can also try using a different USB cable and another wall adapter.
-
-<hr />
 
 ### Why is my station showing offline?
 
-Assuming that the WG1200 Gateway unit is powered on, this may be due to a bad internet connection or bad RF connection between the D1 Gateway and the outdoor station.  
+Assuming that the WG3000 Gateway unit is placed outdoors facing south, this may be due to a bad RF connection between the Pulse Gateway and the outdoor station.
```
Comment (gkoumanos) [inline docs/wxm-devices/Pulse-bundle/Pulse-troubleshooting.mdx:106]: `Pulse Gateway` multiple names for the same thing causes confusion

File: docs/wxm-devices/Pulse-bundle/pulse-bundle-introduction.mdx [hunk 2]
```diff
@@ -27,28 +30,144 @@ Let's see what are the necessary steps in order to properly assemble, install, c
...
+  defaultValue="metric"
+  values={[
+    { label: 'Metric', value: 'metric', },
+    { label: 'Imperial', value: 'imperial', },
+  ]
+}>
+
+<TabItem value="metric">
+
+Powered via a small solar panel and batteries and communicates with the WG1200 Gateway via RF (Radio Frequencies).
```
Comment (gkoumanos) [inline docs/wxm-devices/Pulse-bundle/pulse-bundle-introduction.mdx:82]: WG1200

File: docs/wxm-devices/Pulse-bundle/pulse-bundle-introduction.mdx [hunk 3]
```diff
@@ -27,28 +30,144 @@ Let's see what are the necessary steps in order to properly assemble, install, c
...
+The maximum distance between this station and the WG1200 is 100m with line of sight.
```
Comment (gkoumanos) [inline docs/wxm-devices/Pulse-bundle/pulse-bundle-introduction.mdx:83]: WG1200

File: docs/wxm-devices/Pulse-bundle/pulse-bundle-introduction.mdx [hunk 4]
```diff
@@ -27,28 +30,144 @@ Let's see what are the necessary steps in order to properly assemble, install, c
...
+- #### Temperature:
```
Comment (gkoumanos) [inline docs/wxm-devices/Pulse-bundle/pulse-bundle-introduction.mdx:84]: I skipped the specs. I assume they are right

File: docs/wxm-devices/Pulse-bundle/pulse-bundle-introduction.mdx [hunk 5]
```diff
@@ -27,28 +30,144 @@ Let's see what are the necessary steps in order to properly assemble, install, c
...
+  - Accuracy: ±15%
+
+- #### Pressure:
+
+  - Range: 300 ~ 1100 hPa (equivalent to +9000m ~ -500m above / below sea level)
+  - Accuracy: ±3.9Pa
+  - Resolution: 0.06Pa
+
+- #### Sensor :
+  - Reporting Interval: 16 seconds
```
Comment (gkoumanos) [inline docs/wxm-devices/Pulse-bundle/pulse-bundle-introduction.mdx:122]: Not 16. Most probably 20.

File: docs/wxm-devices/Pulse-bundle/pulse-contents.mdx [hunk 6]
```diff
@@ -77,20 +77,19 @@ The outdoor sensor is responsible for:
...
 
 <div class="control">
-  <img src={useBaseUrl("/img/home/WG1200-gateway-side.png")} style={{ width: '50%', height: 'auto' }} />
+  <img src={useBaseUrl("/img/wxm-devices/pulse-bundle/pulse.png")} style={{ width: '60%', height: 'auto' }} />
 </div>
-The WG1200 Gateway (D1 WiFi) is the device responsible for receiving weather measurements from the outdoor sensor and transmitting them via WiFi to the WeatherXM network.
+The WG3000 Gateway is the device responsible for receiving weather measurements from the outdoor sensor and transmitting them via cellular connection  to the WeatherXM network.
 
-The D1 Gateway is advised to be placed indoors, in an environment of relatively constant temperature (ideally within a daily range of &#60;4°C/7°F for proper functioning of the barometer).
+The Pulse Gateway is advised to be placed outdoors, ideally on a the same pole as the Weather Station, to ensure the best possible connection to the cellular network and with the Weather Station.
```
Comment (gkoumanos) [inline docs/wxm-devices/Pulse-bundle/pulse-contents.mdx:91]: Name

File: static/img/wxm-devices/pulse-bundle/pulse-location.jpeg
Comment (gkoumanos) [inline static/img/wxm-devices/pulse-bundle/pulse-location.jpeg:1]: Why not Manhattan map? Also greek text???

**Review by gkoumanos — CHANGES_REQUESTED** [review]
Please review my comments

File: docs/wxm-devices/Pulse-bundle/Pulse-connect-and-claim.mdx [hunk 7]
```diff
@@ -1,55 +1,50 @@
...
 
 <div class="explanation">
-  <img src={useBaseUrl("/img/wxm-devices/D1-bundle/D1-set_freq.png")} />
+  <img src={useBaseUrl("/img/wxm-devices/pulse-bundle/pulse.png")} />
   <span>
-The WG1200 Gateway's frequency must be set to match the frequency of your weather station so it can listen for the weather data it transmits. There are 2 frequencies available:
+The WG3000 Gateway's frequency must be set to match the frequency of your weather station so it can listen for the weather data it transmits. There are 2 frequencies available:
 - Model A: 868 MHz (default)
 - Model B: 915 MHz
+- Model C: 921 MHz
```
[the remark this answers is not in the document]
Comment (ipanagiv) [reply docs/wxm-devices/Pulse-bundle/Pulse-connect-and-claim.mdx]: The pulse has the frequency available. With whom should we confirm?

[the remark this answers is not in the document]
Comment (gkoumanos) [reply docs/wxm-devices/Pulse-bundle/Pulse-connect-and-claim.mdx]: @NikosTsiligaridis do we have WS1001 in 921 too? Doubt it...

File: docs/wxm-devices/Pulse-bundle/pulse-bundle-introduction.mdx (hunk 3 above)
Comment (ipanagiv) [reply docs/wxm-devices/Pulse-bundle/pulse-bundle-introduction.mdx]: thanks!

File: static/img/wxm-devices/pulse-bundle/pulse-location.jpeg
Comment (ipanagiv) [reply static/img/wxm-devices/pulse-bundle/pulse-location.jpeg:1]: done thanks

File: docs/home.mdx [hunk 8]
```diff
@@ -54,7 +54,7 @@ WeatherXM weather stations are the world’s first to natively support Web3 tech
       <img src={useBaseUrl("/img/wxm-devices/pulse-bundle/pulse_bundle.png")} />
       <div class="device-info" >
         <h4>WeatherXM Pulse bundle (WB3000)</h4>
-        <p>The WB3000 is a bundle product that combines the WS1001 weather station with WeatherXM WG3000 Cellurar Gateway.</p>
+        <p>The WB3000 is a bundle product, that combines the WS1001 weather station with our PULSE 4G - LTE gateway (WB3000). </p>
```
[the remark this answers is not in the document]
Comment (ipanagiv) [reply docs/home.mdx]: i got it from the shop: https://weatherxm.com/product/wb3000-4g/

File: docs/wxm-devices/Pulse-bundle/pulse-bundle-introduction.mdx (hunk 5 above)
Comment (gkoumanos) [reply docs/wxm-devices/Pulse-bundle/pulse-bundle-introduction.mdx]: 30

File: docs/home.mdx (hunk 8 above)
[the remark this answers is not in the document]
Comment (gkoumanos) [reply docs/home.mdx]: @ipanagiv  model in parenthesis is WG... not WB...

**Review by PavlosTze — APPROVED** [review]

File: docs/wxm-devices/pulse-bundle/introduction.mdx [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+  defaultValue="metric"
+  values={[
+    { label: 'Metric', value: 'metric', },
+    { label: 'Imperial', value: 'imperial', },
+  ]
+}>
+
+<TabItem value="metric">
+
+Powered via a small solar panel and batteries and communicates with the WG3000 Gateway via RF (Radio Frequencies).
```
Comment (gkoumanos) [inline docs/wxm-devices/pulse-bundle/introduction.mdx:82]: Called just WG3000 in the other tab

File: docs/wxm-devices/pulse-bundle/connect-and-claim.mdx [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+<hr/>
+### Enter your Claiming key
+
+<div class="explanation">
+    <img style={{width:"calc(100%*1/3 - 10px)"}} src={useBaseUrl("/img/wxm-devices/pulse-bundle/pulse-claiming-key.jpeg")} />
+    <div style={{width:"calc(100%*2/3 - 10px)"}} >
+
+    #### Navigate to W3000 Gateway and reveal the claiming key
+
+    Navigate to "Devise Settings" menu by pressing the arrow keys on your WG3000 right on the screen and select the "Claim Device" option, a QR code will appear.
```
Comment (gkoumanos) [inline docs/wxm-devices/pulse-bundle/connect-and-claim.mdx]: typo: Devise. Please copy paste all docs in spellchecker

File: docs/wxm-devices/pulse-bundle/maintenance.mdx [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+The maintenance of the weather station is an important process, which should be conducted meticulously over the suggested period of time.
+This process includes the cleaning of certain parts of the outdoor unit and the replacement of batteries.
+The maintenance process is summarised in the following steps:
+
+<div class="explanation">
+  <img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/maintenance.svg")} />
+  <div>
+    <h4>Instructions for opening the rain gauge</h4>
+    <ol>
+      <li>Clean the rain gauge at least once every 3 months.</li>
```
Comment (gkoumanos) [inline docs/wxm-devices/pulse-bundle/maintenance.mdx:21]: Numbered list does not correspond to the number on the graphic. Is this reused somehow for D1 too? If not replicate change there too

**Review by PavlosTze — APPROVED** [review]

File: docs/wxm-devices/pulse-bundle/pulse-contents.mdx [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+      <li>Wind speed sensor</li>
+      <li>Weather station</li>
+      <li>Wind vane</li>
+      <li>Bracket (for WG3000)</li>
+      <li>Bracket (for pole mounting)</li>
+      <li>Jubilee clips (x2)</li>
+      <li>Screws for securing the bracket to WG3000 (x3)</li>
+      <li>Screws and fischer wall plugs (x2) </li>
+      <li>WG3000 Gateway</li>
+      <li>Power supply cable (USB-C)</li>
```
[the remark this answers is not in the document]
Comment (PavlosTze) [reply docs/wxm-devices/pulse-bundle/pulse-contents.mdx]: Yes, we do the same on [M5 ](https://docs.weatherxm.com/wxm-devices/wifi-m5-bundle/m5-contents) and [D1](https://docs.weatherxm.com/wxm-devices/D1-bundle/D1-contents).

File: static/img/wxm-devices/pulse-bundle/contents-included-pulse.png
[the remark this answers is not in the document]
Comment (PavlosTze) [reply static/img/wxm-devices/pulse-bundle/contents-included-pulse.png:1]: Let's move on without that in order not to lose time, as it's minor, and I informed the designer to look into it and give us a new sketch whenever they can. But, good catch :)

[the remark this answers is not in the document]
Comment (PavlosTze) [reply static/img/wxm-devices/pulse-bundle/contents-included-pulse.png:1]: Nevermind, fixed it we got the new icon :)

**Review by gkoumanos — APPROVED** [review]
LGTM

**Pull request merged**